scripts/paged_speculative_inference.py

```diff
@@ -38,6 +39,13 @@
     default=None,
     help="Path to the checkpoint containing speculator weights (single .pth file, not HF weights)",
 )
+parser.add_argument(
+    "--speculator_source",
+    type=str,
+    default=None,
+    choices=["hf"],
+    help="Source format of speculator weights",
```

Comment:
I think this parameter is still the way to decide whether to download weights from hf or read them locally off the disk right?

Reply:
yes, so because we don't yet have remote weight support in `fms` get_model, this will currently try to get the local weights using `get_model`, if it cannot, it will fall to huggingface from_pretrained

Reply:
updated help parameter for this argument in latest commit